resolwe_bio/docker_images/dnaseq/README.md

```diff
@@ -1,14 +1,14 @@
 # Docker image for processes using DNA-Seq tools
 
-The image is based on the [`docker.io/resolwebio/common`](
-https://hub.docker.com/r/resolwebio/common/) image, derived from the
-`ubuntu-18.04` version of [`docker.io/resolwebio/base`](
-https://hub.docker.com/r/resolwebio/base/) image.
+The image is based on the [`public.ecr.aws/genialis/common`](
```

Comment:
The `Dockerfile` says it's based on `public.ecr.aws/genialis/resolwebio/common:2.8.0` (notice the missing resolwebio). Path in the [] is correct, though.